movies-manager/src/index.tsx

```diff
@@ -0,0 +1,4 @@
+import React, { Component } from "react";
```

Comment:
Component is unused. You can setup tslint for it.

Reply:
ts lint will be installed in task-5, as was discussed previously, but I will remove it

Reply:
done